html5ever/tests/serializer.rs

```diff
@@ -165,15 +165,15 @@ test!(noframes_literal, r#"<noframes>(x & 1) < 2; y > "foo" + 'bar'</noframes>"#
 
 test!(pre_lf_0, "<pre>foo bar</pre>");
 test!(pre_lf_1, "<pre>\nfoo bar</pre>", "<pre>foo bar</pre>");
-test!(pre_lf_2, "<pre>\n\nfoo bar</pre>");
+test!(pre_lf_2, "<pre>foo bar</pre>");
```

Comment:
As written, this is identical to pre_lf_0. Instead, we should do `test!(pre_lf_2, "<pre>\n\nfoo bar</pre>", "<pre>\n\nfoo bar</pre>");` (and same for the other modified tests).